/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/internal/pycore_ceval.h: 121 - 121
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/internal/pycore_ceval.h: 134 - 134
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/internal/pycore_ceval.h: 143 - 143
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/abstract.c: 2603 - 2629
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/abstract.c: 2635 - 2639
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/Objects/abstract.c: 2645 - 2663
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/internal/pycore_pyerrors.h: 23 - 23
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/object.h: 220 - 220
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/object.h: 234 - 234
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/object.h: 243 - 243
--------------------------------------------------------------------------------

/tmp/python-build.20240827235536.29272/Python-3.12.0/./Include/object.h: 701 - 706
--------------------------------------------------------------------------------

